docs/data/base/components/select/select.md

```diff
@@ -0,0 +1,91 @@
+---
+product: base
+title: React Select unstyled component and hook
+components: SelectUnstyled, MultiSelectUnstyled, OptionUnstyled, OptionGroupUnstyled
+githubLabel: 'component: select'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#combobox
+---
+
+# Select
+
+<p class="description">Select components are used for choosing options from a list.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+MUI Base offers two components that aim to replace the native `<select>` tag: SelectUnstyled and MultiSelectUnstyled.
+
+## The SelectUnstyled component
+
+```tsx
+import SelectUnstyled from '@mui/base/SelectUnstyled';
+```
+
+### Basic usage
+
+{{"demo": "UnstyledSelectSimple.js"}}
+
+The `SelectUnstyled` is a component that accepts generic props.
+Due to Typescript limitations, this may cause unexpected behavior when wrapping the component in `forwardRef` (or other higher-order components).
```

Comment:
```suggestion
Due to TypeScript limitations, this may cause unexpected behavior when wrapping the component in `forwardRef` (or other higher-order components).
```